### PR TITLE
Fix containerized generation

### PR DESCRIPTION
**Pull request by cblecker**
Containerized generation wasn't working under podman. This updates it to work under both :)

### CODE REVIEW

File: hack/verify.sh [hunk 1]
```diff
@@ -32,7 +32,7 @@ fi
 # Excluded check patterns are always skipped.
 EXCLUDED_PATTERNS=(
   "verify-all.sh"                # this script calls the make rule and would cause a loop
-  "verify-*-dockerized.sh"       # Don't run any scripts that intended to be run dockerized
+  "verify-*-containerized.sh"    # Don't run any scripts that intended to be run dockerized
```
Comment (MadhavJivrajani) [inline hack/verify.sh:35]:
```suggestion
  "verify-*-containerized.sh"    # Don't run any scripts that intended to be run containerized
```
😅

**Review by MadhavJivrajani — COMMENTED** [review]
One comment, LGTM otherwise, thanks!

File: hack/verify.sh (hunk 1 above)
Comment (cblecker) [reply hack/verify.sh]: oops! fixed haha

**Review by MadhavJivrajani — APPROVED** [review]
/lgtm
/approve
Thanks!

**Pull request merged**